native/include/mixpanel/mixpanel.hpp

```diff
@@ -14,11 +14,16 @@
 #include <stdexcept>
 #include <memory>
 #include "./value.hpp"
+#include "../../tests/gtest/include/gtest/gtest_prod.h"
 
 #if defined(_MSC_VER)
 #    pragma warning( disable : 4290 )
 #endif
 
+class MixpanelNetwork_RetryAfter_Test;
+class MixpanelNetwork_BackOffTime_Test;
+class MixpanelNetwork_FailureRecovery_Test;
+
```

Comment:
I believe you can remove these 4 added lines, now that lines 236–238 have the `::`s.